### ISSUE TITLE
Is this possible to use compressed books?

### ISSUE THREAD

**Issue by eirenik0** [futurepress/epubjs-angular-reader#28]
Hi everyone, I need to use compressed *.epub books, but when I try open it shown `Zip lib not loaded`.
Even when I add `zip.min.js` it doesn't load properly.

Someone knows a solution? Thank you.

**Comment by Allaoua9** [comment]
Yes, i tried it too but it doesn't work either. Any solution .please ?
